src/sentry/workflow_engine/models/data_source.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    DefaultFieldsModel,
+    FlexibleForeignKey,
+    region_silo_model,
+)
+
+
+@region_silo_model
+class DataSource(DefaultFieldsModel):
+    __relocation_scope__ = RelocationScope.Organization
+
+    class Type(models.IntegerChoices):
+        SNUBA_QUERY_SUBSCRIPTION = 1
```

Comment:
I assume this is what we're using type for here?

Reply:
👍 - i do think we'll have more types, but we can add as needed.